src/modules/billing/routes/invoices.js

```diff
@@ -23,3 +23,19 @@ exports.patchInvoice = {
     }
   }
 };
+
+exports.resetIsFlaggedForRebilling = {
+  method: 'PATCH',
+  path: '/water/1.0/billing/invoices/rebillingflag/{batchId}',
```

Comment:
You need to add the invoiceId as another parameter to only reset the flag for that specific invoice